### ISSUE TITLE
[Feature request] Backward search of objects

### ISSUE THREAD

**Issue by obodeldog** [ioBroker/ioBroker.javascript#183]
If you use an object in many places (scripts, scenes, vis pages, etc.), it would be desirable to have an overview where the object is used everywhere.

**Comment by AlCalzone** [comment]
This could be hard or even next to impossible to do. If its used in an adapter, one would have to search the code of that adapter and its modules. 

Also it might be that the id is not in cleartext: 
`getObject("adapter.0" + somethingCalculated + ".state")`

**Comment by GermanBluefox** [comment]
We must try. Of course not all cases will be covered, but something similar (99,5% of usecases) will be possible.

**Comment by OstfrieseUnterwegs** [comment]
Could it be an option to capture those references as key/value whenever an object like view, script, scene, enum, text-to-command is being created/modified/deleted ? Then only that table needs to be searched.

**Comment by Apollon77** [comment]
Yes, but this is a lot of data to gather during normal system runtime ... so if, it needs to make sure that there are no performance issues caused by collecting those data ... just for such a rare usecase ...
